common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -33,6 +33,7 @@ import android.support.v4.media.MediaMetadataCompat
 import android.support.v4.media.session.MediaControllerCompat
 import android.support.v4.media.session.MediaSessionCompat
 import android.support.v4.media.session.PlaybackStateCompat
+import android.util.Log
```

Comment:
We generally try to keep logging out of production code.

Reply:
Got it - removed!